src/main/java/org/spongepowered/mod/SpongeGame.java

```diff
@@ -97,7 +101,24 @@ public String getImplementationVersion() {
 
     @Override
     public MinecraftVersion getMinecraftVersion() {
-        throw new UnsupportedOperationException(); // TODO
+	    MinecraftVersion mcvToReturn;
+
+	    // Get the MC version and protocol version from the MC server, then parse it so we can use it earlier
+	    ServerStatusResponse.MinecraftProtocolVersionIdentifier protocolVersionInfo = MinecraftServer.getServer().getServerStatusResponse().getProtocolVersionInfo();
+	    String name = protocolVersionInfo.getName();
+	    int protocolVersion = protocolVersionInfo.getProtocol();
+	    double numberVersion = Double.valueOf(name);
+
+	    // If the version is less than 1.7 it is considered legacy and must be handled differently
+	    if (numberVersion < 1.7) {
```

Comment:
Our implementation will never support legacy so there is no need to check for 1.7 and under.